src/Analysis/Ast/Impl/Types/Definitions/IBuiltinPythonModule.cs

```diff
@@ -33,5 +33,7 @@ namespace Microsoft.Python.Analysis.Types {
     /// </summary>
     public interface IBuiltinsPythonModule : IPythonModule {
         IMember GetAnyMember(string name);
+
+        void ResetAst();
```

Comment:
Dangerous to have this on a public interface. Safer to do internal method.

Reply:
Internal meaning I try casting to the real thing and go from there?